src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
 public class WorkWithFile {
```

Comment:
The class 'WorkWithFile' could have a more specific name that reflects its purpose, like 'TransactionReportGenerator'.